scripts/Editor/DllManipulatorEditor.cs

```diff
@@ -313,7 +303,32 @@ private void DrawOptions(DllManipulatorOptions options)
             GUI.enabled = guiEnabledStack.Pop();
         }
 
-        private void DrawList<T>(IList<T> elements, Func<int, T> drawElement, bool canAddNewElement, Func<T> getNewElement)
+        /// <summary>
+        /// Will search for all managed assemblies and store them in <see cref="_allKnownAssemblies"/>.
+        /// Excludes assemblies starting with <see cref="DllManipulator.IGNORED_ASSEMBLY_PREFIXES"/>
+        /// </summary>
+        private void RefreshAllKnownAssemblies()
+        {
+            var playerCompiledAssemblies = CompilationPipeline.GetAssemblies(AssembliesType.Player)
+                .Select(a => Path.GetFileNameWithoutExtension(a.outputPath));
+
+            var editorCompiledAssemblies = CompilationPipeline.GetAssemblies(AssembliesType.Editor)
+                .Select(a => Path.GetFileNameWithoutExtension(a.outputPath));
+
+            var assemblyAssets = Resources.FindObjectsOfTypeAll<PluginImporter>()
+                .Where(p => !p.isNativePlugin)
+                .Select(p => Path.GetFileNameWithoutExtension(p.assetPath));
+
+            _allKnownAssemblies = playerCompiledAssemblies
+                .Concat(editorCompiledAssemblies)
+                .Concat(assemblyAssets)
+                .Where(a => !DllManipulator.IGNORED_ASSEMBLY_PREFIXES.Any(a.StartsWith))
```

Comment:
To be exact, this wouldn't be `allKnownAssemblies` if we exclude some here. So maybe rename that to sth like `possibleTargetAssemblies`?

Reply:
makes sense, changed to `possibleTargetAssemblies`